packages/documentation-framework/components/example/example.css

```diff
@@ -101,32 +67,32 @@
 }
 
 .pf-v6-c-badge.ws-beta-badge {
-/*   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-v6-global--BackgroundColor--100); */
-/*   --pf-v6-c-badge--m-unread--Color: var(--pf-v6-global--primary-color--100); */
-/*   border: var(--pf-v6-global--BorderWidth--sm) solid var(--pf-v6-global--primary-color--100); */
+   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-t--global--background--color--floating--default);
+   --pf-v6-c-badge--m-unread--Color: var(--pf-t--global--color--status--unread--default--default);
+   border: var(--pf-t--global--border--width--regular) solid var(--pf-t--global--border--color--nonstatus--blue--default);
 }
 
 .ws-prop-required {
-/*   color: var(--pf-v6-global--danger-color--100); */
+  color: var(--pf-t--global--text--color--status--danger--default);
 }
 
 .ws-full-page-utils {
   position: fixed;
   right: 0;
   bottom: 0;
   padding: var(--pf-t--global--spacer--lg);
-/*   z-index: var(--pf-v6-global--ZIndex--2xl); */
+  z-index: var(--pf-t--global--z-index--600);
 }
 
 .ws-full-page-utils::before {
   position: absolute;
   inset: 0;
   content: "";
-/*   background-color: var(--pf-v6-global--BackgroundColor--100); */
+  background-color: var(--pf-t--global--background--color--floating--default);
   opacity: 0.8;
-/*   box-shadow: var(--pf-v6-global--BoxShadow--sm); */
+  box-shadow: var(--pf-t--global--box-shadow--color--sm);
 }
 
 .pf-v6-theme-dark .ws-theme-switch-full-page::before {
```

Comment:
Do you know where this is on the site? Poked around some on the PR preview site and couldn't find it.

Reply:
They are the dark theme switchers in the bottom right corner of the full page examples:
https://patternfly-org-pr-4081-site.surge.sh/components/page/react-demos/sticky-section-group/

Reply:
Hmm... I don't see it in the markup on that page. IIRC I renamed `.ws-theme-switch-full-page` to `.ws-full-page-utils` in https://github.com/patternfly/patternfly-org/commit/fc1f9bacab352cf2db7418ae98b2f3c98cd47302, and probably missed removing this selector.

Reply:
I'll take a look